src/DeviceInput/Implementations/webDeviceInputSystem.ts

```diff
@@ -609,6 +622,10 @@ export class WebDeviceInputSystem implements IDeviceInputSystem {
                         this._unregisterDevice(DeviceType.Touch, deviceSlot);
                     }
                 }
+                // Clear all active touches
+                while (this._activeTouchIds.length > 0) {

```

Comment:
so, not `.length` will work, but if you want to do it this way, you can always have ano-op loop like this:

```suggestion
                while (this._activeTouchIds.pop() !== undefined) { }

```